monai/transforms/intensity/array.py

```diff
@@ -967,6 +968,18 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
         img = scalar(img)
         return img
 
+    def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
+        """
+        Apply the transform to `img`.
+        """
+        if self.channel_wise:
+            for i, d in enumerate(img):
+                img[i] = self._normalize(img=d)  # type: ignore
```

Comment:
in this case we should probably introduce `axis` to percentile for multichannel computation

Reply:
Sure, added `dim` in the latest commit.

Thanks.